frontend/src/pages/home/home.tsx

```diff
@@ -287,17 +249,15 @@ export default class HomePage extends React.Component<HomePageProps, HomePageSta
                     {this.renderCardSection(comboCardData)}
                     <div className="justify-center">
                         <div className="notice-container">
-                            <div className="notice-content">
+                            <div className="notice-content" style={{borderRadius: 0}}>
                                 <p>These are all the messages we managed to collect!</p>
-                                <p>Welcome back, Haato. おかえり、はあと。</p>
-                                <p style={{ fontSize: 12 }}>If you find any problems with the website, or if you would like to report a message, please contact us at manotomo@googlegroups.com</p>
+                                <p>Tweet at #TOBEDECIDED to send us a message!</p>
+                                <p style={{fontSize: 12}}>If you find any problems with the website, or if you would like to report a message, please contact us at manotomo@googlegroups.com or at webmaster@manotomo.com</p>
+                                <p style={{fontSize: 12}}>This is not an official Hololive site. We are just a group of fans</p>
```

Comment:
This sentence ended weird. "...group of fans supporting Aqua!"?

Reply:
gotcha